include/uapi/sound/sof/eq.h

```diff
@@ -71,10 +71,12 @@ struct sof_eq_fir_coef_data {
 	int16_t coef[]; /* FIR coefficients */
 } __packed;
 
-/* In the struct above there's two words (length, shift) before the actual
- * FIR coefficients. This information is used in parsing of the config blob.
+/* In the struct above there's two 16 bit words (length, shift) and four
+ * reserved 32 bit words before the actual FIR coefficients. This information
+ * is used in parsing of the configuration blob.
  */
-#define SOF_EQ_FIR_COEF_NHEADER 2
+#define SOF_EQ_FIR_COEF_NHEADER \
```

Comment:
@singalsu is this used in the driver? If not, we do need this here?

Reply:
It's not needed by driver but this is how the user space could use to create & parse the EQ blob.

Reply:
The uapi should contain headers that are needed by user-space apps that use the driver and what distros should include in their kernel header packages. There are many exceptions to these, but it's good for us to follow the generic guidance here. Otherwise distros would need to sync SOF headers for a specific kernel release manually. Now they can just take the headers from matchin kernel's uapi folder.